src/styles/blocks/_button.scss

```
@@ -0,0 +1,42 @@
+.button {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  
+  box-sizing: border-box;
+  width: 100%;
+  height: 56px;
+
+  text-decoration: none;
+  color: $basic-color;
+  background: #fff;
```

don't forget to create variables for all colors
```suggestion
  background: $c-white;
```